src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -1414,3 +1416,11 @@ def test_osd_activate_datadevice_dbdevice(self, _run_cephadm, cephadm_module: Ce
                               '--config-json', '-', '--osd-fsid', 'uuid'],
                           stdin=mock.ANY, image=''),
             ]
+
+    def host_rm_last_admin(self, cephadm_module: CephadmOrchestrator):
+        with pytest.raises(OrchestratorError):
+            with with_host(cephadm_module, 'test', refresh_hosts=False, force=False):
+                pass
```

Comment:
```suggestion
                c = cephadm_module.remove_host('test')
                [out] = wait(cephadm_module, c)
```
?

Reply:
I tested it locally here. `force=False` makes remove_host raise an exception that is catched by pytest.raises. At least this was the plan